media/js/base/mozilla-fxa-link.js

```diff
@@ -43,12 +86,21 @@ if (typeof window.Mozilla === 'undefined') {
             return;
         }
 
+        var userVer = parseFloat(client._getFirefoxVersion());
+        var useUITourForFxA = userVer > 78 && Mozilla.UITour;
         var fxaSigninLink = document.querySelectorAll('.js-fxa-cta-link');
 
         for (var i = 0; i < fxaSigninLink.length; i++) {
             var link = fxaSigninLink[i];
             var mozillaOnlineLink = link.getAttribute('data-mozillaonline-link');
 
+            if (useUITourForFxA) {
+                // intercept the flow and submit the form using the UITour API instead.
+                // In the future we should fully migrate to this API for Firefox Desktop login.
+                link.addEventListener('auxclick', this.interceptFxANavigation);
+                link.addEventListener('click', this.interceptFxANavigation);
+                return;
```

Comment:
The `return` statement here is preventing the UITour even from being bound to all but the first link in the page.